internal/faustjs.org/package.json

```
@@ -3,13 +3,12 @@
   "private": true,
   "scripts": {
     "docusaurus": "docusaurus",
-    "start": "docusaurus start",
+    "dev": "docusaurus start",
```

Shouldn't this script have the changelogs downloaded as well? Such as? I'm realizing this may need to be added as a second ticket, since the changelogs are only configured for the legacy site at the moment
```suggestion
    "start": "npm run downloadChangelogs && docusaurus serve",
```

---

Yes! They should, but we don't have that setup yet. [MERL-577](https://wpengine.atlassian.net/browse/MERL-577) should take care of that work.

---

Oh perfect! Thank you so much Blake!